src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql

```diff
@@ -1,509 +1,830 @@
-{% set periods = ["BOY", "BOY->MOY", "MOY", "MOY->EOY", "EOY"] %}
+{% set periods = ["BOY", "BOY->MOY", "MOY", "MOY->EOY", "EOY"] %}  -- Force expected assessments
 
+-- Import CTEs
 with
-    union_relations as (
-        select
-            bss.surrogate_key,
-            bss.student_primary_id,
-            bss.academic_year,
-            bss.benchmark_period as period,
-            bss.client_date,
-            bss.sync_date,
-            bss.reporting_class_id,
-            bss.reporting_class_name,
-            bss.assessing_teacher_staff_id,
-            bss.assessing_teacher_name,
-            bss.official_teacher_staff_id,
-            bss.official_teacher_name,
-            bss.assessment_grade,
+    iready_roster as (
+        select *
+        from {{ ref("base_iready__diagnostic_results") }}  -- Lists eligible 3rd and 4th students to take DIBELS based on iReady BOY placement
+        where
+            test_round = 'BOY'
+            and student_grade in ('3', '4')
+            and rn_subj_round = 1
+            and overall_relative_placement_int <= 2
+            and academic_year = '2022-2023'

```

Comment:
hard-coded year

Reply:
academic_year is in that format... i will swap it to academic_year_int

Reply:
actually no. i cant change it to int... let me left it

Reply:
i had to do this: left(academic_year,4) = cast({{ var("current_academic_year") }} as string) or i would get a dbt build error for wrong types